src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                    .flatMap(str -> Arrays.stream(str.split(",")))
+                    .mapToInt(Integer::parseInt)
+                    .filter(number -> number % 2 == 0).min()
+                    .orElseThrow(() -> 
+                        new RuntimeException("Can't get min value from list: " 
+                                            + numbers.toString()));
```

Comment:
```suggestion
                                            + numbers));
```
toString() called automatically